arches_for_science/templates/views/components/plugins/init-workflow.htm

```diff
@@ -1,13 +1,97 @@
-<div class="workflow-select-plugin">
-    <div class="workflow-select-card-container">
-        <!-- ko foreach: {data: workflows, as: 'workflow'} -->
-        <a data-bind="attr: {href: workflow.url, title: workflow.name}" href="#"><div data-bind="style: {'background-color': workflow.bgColor}" class="workflow-select-card">
-            <h4 class="workflow-select-title" data-bind="text: workflow.name"></h4>
-            <div data-bind="style: {'background-color': workflow.circleColor}" class="workflow-select-wf-circle">
-                <span><i data-bind="attr:{class: ('fa '+workflow.icon +' workflow-select-wf-icon')}"></i></span>
+{% load i18n %} 
+
+<div>
+    <!-- Modal -->
+    <!-- ko if: shouldShowIncompleteWorkflowsModal() -->
+    <div 
+        style="
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            background: rgba(0, 0, 0, .5);
+            z-index: 1;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            cursor: pointer;
+        "
+    >
+        <div 
+            style="background-color: white; width: 1000px; max-height: 480px; border-radius: 4px; cursor: default; overflow: hidden;"
+        >
+            <div style="width: 100%; display: flex; justify-content: flex-end; padding: 10px 12px;">
+                <i 
+                    class="fa fa-times" style="font-size: 20px; cursor: pointer;"
+                    data-bind="click: function(){ $data.shouldShowIncompleteWorkflowsModal(false); }"
+                ></i>
             </div>
-            <p data-bind="text: workflow.desc" class="workflow-select-desc"></p>
-        </div></a>
-        <!-- /ko -->
+            <div style="display: flex; flex-direction: column; align-items: center;">
+                <h2>{% trans 'You have incomplete workflows' %}</h2>
+                <p>
+                    {% trans 'Please either complete or delete the workflows listed below.' %}
+                </p>
+            </div>
+
+            <div style="padding: 40px 80px; display: flex; justify-content: center;">
+                <table class="table table-striped" style="border: 1px solid #ddd;">
+                    <thead>
+                        <tr style="display: flex; justify-content: space-between;">
+                            <th style="width: 219px;">
+                                {% trans 'Created' %}
+                            </th>
+                            <th style="width: 400px;">
+                                {% trans 'Workflow Name' %}
+                            </th>
+                            <th style="width: 219px;">
```

Comment:
The styles in this modal are really great.

One tiny thing I noticed was header/cell value alignment when there's a scrollbar (e.g. MacOS automatic/always instead of "when scrolling").

Might not be worth addressing, I've just never dealt with this specific issue and wondered if you've seen solutions for it before.
![align](https://github.com/archesproject/arches-for-science/assets/38668450/91b7b97f-16b4-496a-865a-813907de4229)

Reply:
Huh, I believe you but am unable to repro even when enabling the scrollbar to `always`. I'm assuming os versions have something to do with this. I've made some updates to the table that hopefully resolve this ( 🤞 ) but without being able to repro I'm unable to make sure

Reply:
Yeah it's still there, but I'm not worried about it. You might be right it could be some macOS regression or something!